.gitignore

```diff
@@ -0,0 +1,2 @@
+#Files to ignore
+test.md
```

Comment:
- [ ] **Required:** You are required to ignore the `node_modules` DIR

![image](https://user-images.githubusercontent.com/27709832/164063089-50ca1afa-79d3-421a-9f30-e20f6f1c2b88.png)